src/server_manager/web_app/app.ts

```diff
@@ -209,10 +213,14 @@ export class App {
   // Show the DigitalOcean server creator or the existing server, if there's one.
   private enterDigitalOceanMode(accessToken: string) {
     const doSession = this.createDigitalOceanSession(accessToken);
-    const oauthUi = this.appRoot.getDigitalOceanOauthFlow();
     const authEvents = new events.EventEmitter();
     let cancelled = false;
     let activatingAccount = false;
+    const handleOauthFlowCanceled = () => {
+      cancelled = true;
+      this.clearCredentialsAndShowIntro();
+    };
+    const oauthUi = this.appRoot.getDigitalOceanOauthFlow(handleOauthFlowCanceled);
```

Comment:
We are not running the Oauth flow here. It happened already. We are verifying the state of the account. The current name gives the impression we are running the Oauth flow. Let's rename it to better express what we are doing. For example `cancelAccountStateVerification` or `cancelWaitForActiveAccount`